LayoutTests/platform/glib/TestExpectations

```diff
@@ -1181,9 +1182,6 @@ media/media-source/media-managedmse-memorypressure.html [ Timeout ]
 media/media-source/media-managedmse-memorypressure-inactive.html [ Timeout ]
 media/media-source/media-managedmse-eviction.html [ Timeout ]
 
-# No MSE in a worker on glib platforms for now (requires GPU process).
-media/media-source/worker/media-managedmse-worker.html [ Timeout ]
```

Comment:
Is this really passing? I'm a bit surprised because our GPUProcess doesn't handle media yet.

Reply:
It seems so.  See history at https://results.webkit.org/?platform=WPE&platform=GTK&suite=layout-tests&test=media%2Fmedia-source%2Fworker%2Fmedia-managedmse-worker.html.

TBH, I'm surprised too. It passes with my local runs as well.